### ISSUE TITLE
Warning with Angular 13.2.0

### ISSUE THREAD

**Issue by sbusch99** [denniske/ngx-translate-multi-http-loader-demo#1]
You now have to do `npm i --force`
```
npm i --force
npm WARN using --force Recommended protections disabled.
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-translate-multi-http-loader@7.0.1
npm WARN Found: rxjs@7.5.2
npm WARN node_modules/rxjs
npm WARN   rxjs@"~7.5.0" from the root project
npm WARN   11 more (@angular/core, @angular/cdk, @angular/common, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer rxjs@"~7.4.0" from ngx-translate-multi-http-loader@7.0.1
npm WARN node_modules/ngx-translate-multi-http-loader
npm WARN   ngx-translate-multi-http-loader@"^7.0.1" from the root project
npm WARN 
npm WARN Conflicting peer dependency: rxjs@7.4.0
npm WARN node_modules/rxjs
npm WARN   peer rxjs@"~7.4.0" from ngx-translate-multi-http-loader@7.0.1
npm WARN   node_modules/ngx-translate-multi-http-loader
npm WARN     ngx-translate-multi-http-loader@"^7.0.1" from the root project
npm WARN deprecated source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated

```

**Comment by denniske** [comment]
Hi, I just published version 7.0.2 which has less strict dependency versions. Can you check whether it works now?

**Comment by sbusch99** [comment]
Fixed! Thank you :+1:
